packages/IMAPClient-Core.package/ICAccountInfo.class/instance/printOn..st

```diff
@@ -1,4 +1,10 @@
 printing
 printOn: aStream
 
-	aStream nextPutAll: self accountName, ' ', self host, ' ', self port, ' ', self username, ' ', self password, ' ', self ssl
\ No newline at end of file
+	aStream nextPutAll: 
```

Comment:
Use `joinSeparatedBy:` here instead of string, blank, string, blank

Reply:
![image](https://user-images.githubusercontent.com/5970076/61959950-cda36880-afc4-11e9-8969-58e97b992286.png)
Looks like this now... I think this is only slightly better :/
Any better ideas?

Reply:
A newline after every dot?
``` smalltalk
aStream nextPut: ({
    self username.
    self password.
   ...} joinSeparatedBy: '')
```

Reply:
fair :)